Memory/Hooks.cpp

```diff
@@ -183,7 +183,7 @@ void Hooks::Init() {
 
 	// Signatures
 	{
-		void* surv_tick = reinterpret_cast<void*>(FindSignature("48 89 5C 24 10 48 89 74 24 18 55 57 41 56 48 8D 6C 24 ?? 48 81 EC ?? ?? ?? ?? 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 45 ?? 48 8B F9 F3"));
+		void* surv_tick = reinterpret_cast<void*>(FindSignature("48 89 5C 24 ? 48 89 74 24 ? 48 89 7C 24 ? 55 48 8D 6C 24 ? 48 81 EC ? ? ? ? 48 8B 05 ? ? ? ? 48 33 C4 48 89 45 4F 48 8B D9 48 8B 89 ? ? ? ?"));
```

Comment:
Doesn't really call anymore

Reply:
does it matter?

Reply:
> does it matter?
because Module::onTick could trigger unexpectedly